app/config/sculpin_site.yml

```diff
@@ -49,6 +49,9 @@ subspecies:
     phpbenelux:
         common: "PHPBenelux"
         latin: "belgerlandemburgi"
+    phpclasses:
+        common: "PHP Classes"
+        latin: "decuria"
```

Comment:
Names based on nouns should be in the genitive case. That would make this `decuriae`. Are you okay with that change?

Reply:
That would work.  I also had `archetypon` suggested by a Latin-speaking coworker